guides/hosting/installation-updates/enterprise-project-setup.md

```diff
@@ -0,0 +1,190 @@
+# Enterprise project setup
+
+The setup of enterprise systems differs from a normal installation of Shopware. Most enterprise systems are a completely customized store with individual templates and extensions to make the store work the way it should.
+
+This guide is meant to serve as a suffering guide for all those who do not have any experience in this area yet.
```

Comment:
Is "suffering guide" a turn of phrase? It's sounds rather negative. I think guide should suffice.